package.json

```diff
@@ -16,32 +16,32 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^13.3.11",
-    "@angular/common": "^13.3.11",
-    "@angular/compiler": "^13.3.11",
-    "@angular/core": "^13.3.11",
-    "@angular/elements": "^13.3.11",
-    "@angular/forms": "^13.3.11",
-    "@angular/platform-browser": "^13.3.11",
-    "@angular/platform-browser-dynamic": "^13.3.11",
-    "@angular/router": "^13.3.11",
+    "@angular-eslint/schematics": "^14.4.0",
+    "@angular/animations": "^14.2.12",
+    "@angular/common": "^14.2.12",
+    "@angular/compiler": "^14.2.12",
+    "@angular/core": "^14.2.12",
+    "@angular/elements": "^14.2.12",
+    "@angular/forms": "^14.2.12",
+    "@angular/platform-browser": "^14.2.12",
+    "@angular/platform-browser-dynamic": "^14.2.12",
+    "@angular/router": "^14.2.12",
     "@laserfiche/lf-js-utils": "^4.0.3",
     "@laserfiche/lf-repository-api-client": "^1.0.12",
-    "@laserfiche/lf-ui-components": "^13.1.7",
-    "@laserfiche/lf-ui-components-services": "^13.0.5",
+    "@laserfiche/lf-ui-components": "^14.0.0",
+    "@laserfiche/lf-ui-components-services": "^14.0.0",
     "rxjs": "^6.6.0",
     "tslib": "^2.3.1",
     "zone.js": "^0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^13.3.9",
+    "@angular-devkit/build-angular": "^14.2.10",
     "@angular-eslint/builder": "13.5.0",
```

Comment:
can @angular-eslint/builder, @angular-eslint/eslint-plugin, @angular-eslint/eslint-plugin-template, @angular-eslint/template-parser be upgraded to 14?
upgrade @types/node to 16